user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_VHDX.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: Using_a_pre_installed_DataMiner_VHDX
+---
+
+# Using a pre-installed DataMiner Virtual Hard Disk
+
+You can download a Virtual Hard Disk (VHDX) with DataMiner pre-installed to immediately get started.
+This VHDX comes out-of-the box with a [locally hosted Cassandra Cluster and OpenSearch, running on Windows Subsystem for Linux (WSL)](xref:Local_database_on_WSL).
+
+> [!NOTE]
+> Running an out-of-the box deployment with the [pre-installed DataMiner Virtual Hard Disk](xref:Using_a_pre_installed_DataMiner_VHDX) should only be used for test and staging environments when sufficient resources have been given to the Virtual Machine. Consider migrating to [Storage as a Service (STaaS)](xref:STaaS) or [configure dedicated clustered storage](xref:Configuring_dedicated_clustered_storage) if you intend to use it in production.
+
```

Comment:
First the user will need to enable nested virtualization on their VM.

For Hyper-V this can be done by opening an elevated Powershell windows and run:
`Set-VMProcessor -VMName <VMName> -ExposeVirtualizationExtensions $true`

Reply:
The first time they spin up the system with the disk they will have to create an Administrator account. When they log in with the user, a script will be started. It's important that they don't exit this script as this will configure their system with unique values.

The script will ask for their password to create a scheduled task to run at system start-up to start WSL at boot.